.github/workflows/pr-python.yml

```diff
@@ -9,14 +9,16 @@ on:
 
 env:
   AWS_REGION: us-east-1
+  #copied the core repo sha from here:https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/.github/workflows/test.yml#L9
+  CORE_REPO_SHA: ece57231cde5bb5656775a62f027cab0b1c76fce
```

Comment:
Is this still necessary?

Reply:
this env variable would be necessary to test against the specific commit other than `main` and is in a format the tests to follow the format of upstream